cli/tests/pull.bats

```diff
@@ -329,91 +216,6 @@ function add_incompatible_package() {
 
-# bats test_tags=pull:add-system-flag
-# pulling an environment without packages for the current platform
-#should fail with an error
-@test "pull environment inside the same environment without the '--force' flag" {
-  export FLOX_FEATURES_USE_CATALOG=false
-
-  make_dummy_env "owner" "name"
-  update_dummy_env "owner" "name"
-
-  run "$FLOX_BIN" pull --remote owner/name
-  assert_success
-  run "$FLOX_BIN" pull --remote owner/name
-  assert_failure
-}
-
-# bats test_tags=pull:add-system-flag
-# pulling an environment without packages for the current platform
-@test "pull environment inside the same environment with '--force' flag" {
-  export FLOX_FEATURES_USE_CATALOG=false
-
-  make_dummy_env "owner" "name"
-  update_dummy_env "owner" "name"
-
-  run "$FLOX_BIN" pull --remote owner/name
-  assert_success
-  run "$FLOX_BIN" pull --remote owner/name --force
-  assert_success
-}
-
-# bats test_tags=pull:unsupported:warning
-# An environment that is not compatible with the current ssystem
-# due to the current system missing <system> in `option.systems`
-# AND a package that is indeed not able to be built for the current system
-# should show a warning, but otherwise succeed to pull
-@test "pull v0 unsupported environment succeeds with '--force' flag but shows warning if unable to build still" {
-  make_dummy_env "owner" "name"
-  update_dummy_env "owner" "name"
-  if [ -z "${NIX_SYSTEM##*-linux}" ]; then
-    ENV_FILES_DIR="$MANUALLY_GENERATED/ps_v0_aarch64-darwin"
-    # This response might not be precisely correct if we regenerate because it's
-    # for x86_64-darwin and aarch64-linux, but currently it's the same
-    CATALOG_RESPONSE="$GENERATED_DATA/resolve/darwin_ps_incompatible.json"
-    # TODO: flox uses attr path in errors, while pkgdb uses install ID
-    PACKAGE="darwin.ps"
-  elif [ -z "${NIX_SYSTEM#*-darwin}" ]; then
-    ENV_FILES_DIR="$MANUALLY_GENERATED/glibc_v0_x86_64-linux"
-    # This response might not be precisely correct if we regenerate because it's
-    # for x86_64-linux and aarch64-darwin, but currently it's the same
-    CATALOG_RESPONSE="$GENERATED_DATA/resolve/glibc_incompatible.json"
-    PACKAGE="glibc"
-  else
-    echo "unknown system: '$NIX_SYSTEM'"
-    exit 1
-  fi
-
-  copy_manifest_and_lockfile_to_remote "owner" "name" "$ENV_FILES_DIR"
-
-  _FLOX_USE_CATALOG_MOCK="$CATALOG_RESPONSE" \
-    run "$FLOX_BIN" pull --remote owner/name --force
-  assert_success
-  assert_line --partial "resolution failed: package '$PACKAGE' not available for"
-  assert_line --partial "Migrated the environment to version 1 and included your system but could not"
-
-  run "$FLOX_BIN" list
-  assert_success
-}
-
-# ---------------------------------------------------------------------------- #
-
-# bats test_tags=activate:remote:incompatible
-# activating an incompatible environment should fail gracefully
-@test "activate incompatible environment fails gracefully" {
```

Comment:
nonblocking: how come this one got dropped? Seems like it could regress since we handle system differently for v0 and v1 envs

Reply:
Right, i amended that to a port rather than deleting the test altogether